hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/ReconServerConfigKeys.java

```diff
@@ -39,8 +39,12 @@
       "ozone.recon.http-address";
   public static final String OZONE_RECON_HTTPS_ADDRESS_KEY =
       "ozone.recon.https-address";
-  public static final String OZONE_RECON_KEYTAB_FILE =
-      "ozone.recon.keytab.file";
+  public static final String OZONE_RECON_HTTP_KEYTAB_FILE =
+      "ozone.recon.http.kerberos.keytab.file";
+  public static final String OZONE_RECON_KERBEROS_KEYTAB_FILE_KEY =
+      "ozone.recon.kerberos.keytab.file";
+  public static final String OZONE_RECON_KERBEROS_PRINCIPAL_KEY =
```

Comment:
Move this to org.apache.hadoop.hdds.recon.ReconConfigKeys#ReconConfigKeys since it will be needed by Datanodes as well.